Remove deprecated Guava Dependencies #34

riederm opened the pull request:
fixeds issue #34

riederm committed:
moved to oxygen updatesites and removed deprecated Guava Stuff #34

moved to MoreObjects.firstNonNull and
java.util.Collecionts.emptyIterator

ghaith commented on releng/org.jnario.target/neon-xtext2.12.target:
```diff
@@ -4,15 +4,15 @@
 <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
 <unit id="org.eclipse.m2e.feature.feature.group" version="0.0.0"/>
 <unit id="org.eclipse.draw2d.feature.group" version="0.0.0"/>
-<repository location="http://download.eclipse.org/releases/neon/"/>
```
shouldn't you be creating a new file for oxygen? This is the neon target

riederm replied:
u're right

riederm committed:
introduced oxygen-xtext-2.12.target

restored the neon-xtext2.12.target file to its original state (from
commit 0bcddf)
